styles/styles.css

```
@@ -313,6 +396,27 @@ html {
 
 .button-secondary {
   background-color: #756257;
+  text-decoration: none;
+  border-radius: 4px;
+}
+
+.button.button-secondary:hover {
+  background-color: #615048;
+  border-radius: 4px;
+}
+
+.button-secondary.button:focus {
```

А зачем двойной селектор, чем просто не устроил класс второстепенной кнопки?
Вот так:
```suggestion
.button-secondary:focus {
```